processor/src/main/java/org/mapstruct/ap/internal/model/source/Mapping.java

```diff
@@ -361,6 +374,9 @@ public Mapping copyForInheritanceTo(SourceMethod method) {
             mapping.sourceReference = sourceReference.copyForInheritanceTo( method );
         }
 
+        // TODO... must something be done here? Andreas?
+        mapping.targetReference = targetReference;
```

Comment:
This todo is still here. Sjaak, could you add a test for mapping inheritance to verify that there isn't anything to do here (my guess is that there isn't)? And remove the comment then... :)